src/components/SearchGridFilter.vue

```diff
@@ -147,6 +153,7 @@ export default {
       Object.keys(this.filter).forEach((key) => {
         filter[key] = transformFilterValue(this.filter, key);
       });
+      localStorage.setItem('filter', this.filter);
```

Comment:
Calls to `localStorage` methods should be in a store action or mutation, not inside components.